Source/Scene/ColorMapExpression.js

```diff
@@ -74,10 +74,12 @@ define([
                 return this._default;
             },
             set : function(value) {
-                this._default = value.slice(0);
-                this._styleEngine.makeDirty();
+                if (this._default !== value) {
```

Comment:
This is a reference compare; instead, we would want to check each element in the array.

However, I actually want to change how we expose `map` and `default`.  Let's expose the "compiled" version, which uses `Color` like we do elsewhere (we'll want to do the same thing for color ramp), not the style JSON version.